packages/@okta/vuepress-site/docs/guides/registration-inline-hook/activate-and-enable/index.md

```diff
@@ -20,15 +20,52 @@ The Registration Inline Hook is now set up with a status of active.
 
+### Enabling the Registration Inline Hook
+
+The procedure to enable the Registration Inline Hook is dependent on the type of org you are using: Okta Identity Engine or Okta Classic. Follow the procedure for your specific org.
+
+After you enable the Registration Hook, you are ready to preview and test the example.
+
+#### Enabling the Registration Inline Hook in Okta Identity Engine
+
+<ApiLifecycle access="ie" />
+If you have an Okta Identity Engine org, you must [Enable and configure a profile enrollment policy](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/policies/create-profile-enrollment-policy-sr.htm) to implement a Registration Inline Hook.
+
+> **Note:** Profile Enrollment and Registration Inline Hooks only work with the [Okta Sign-In Widget](/code/javascript/okta_sign-in_widget/) version 4.5 or later.
+
+To associate the Registration Inline Hook with your Profile Enrollment policy:
+
+1. In the Admin Console, go to **Security > Profile Enrollment**.
+
+1. Click the pencil icon to edit the policy and associate it with your Registration Inline Hook.
+
+1. In **Enrollment Settings**, click the More Options icon and then select **Edit**. If not already selected, click **Allowed** in the **For new users** section.
+
+1. Select your hook from the drop-down menu for **Use the following inline hook** under the options for **For new users**. If you have created multiple Registration Inline Hooks, you can see all of them displayed here.
+
+1. Click **Save**.
+
+Your Registration Inline Hook is now configured for Profile Enrollment.
+
+> **Note:** Only one Inline Hook can be associated with your Profile Enrollment policy at a time.
+
+#### Enabling the Registration Inline Hook in Okta Classic
+
+<ApiLifecycle access="ea" />
+If you have an Okta Classic org, you must enable [self-service registration (SSR)](/docs/guides/set-up-self-service-registration/before-you-begin/) to implement a Registration Inline Hook.
+
+> **Note:** Self-service registration and Registration Inline Hooks only work with the [Okta Sign-In Widget](/code/javascript/okta_sign-in_widget/) version 2.9 or later.
+
 To enable the Registration Inline Hook on the self-service registration page:
 
 1. In the Admin Console, go to **Directory** > **Self-Service Registration**.
-2. Click **Edit**.
-3. From the **Extension** field drop-down menu, select the hook you set up and activated previously ("Guide Registration Hook Code") .
-4. Click **Save**.
 
-The Registration Inline Hook is now enabled for self-service registration. For a high-level overview of enabling the Registration Inline hook, see [Registration Inline Hook Reference](/docs/reference/registration-hook/).
+1. Click **Edit**.
+
+1. From the **Extension** field drop-down menu, select the hook you set up and activated previously ("Guide Registration Hook Code") .
```

Comment:
```suggestion
1. From the **Extension** field drop-down menu, select the hook you've set up and activated previously (for example, "Guide Registration Hook Code") .
```

Reply:
This is referring to the hook created in a previous page, so, it's not really an example.